py/examples/table_events_select.py

```diff
@@ -7,7 +7,7 @@
 
 @app('/demo')
 async def serve(q: Q):
-    if q.events.table and q.events.table.select:
+    if q.events.table and q.events.table.select is not None:
```

Comment:
Can the `q.events.select` be an empty string?

Reply:
It can. Anyway, I'm removing this part of condition since empty array is now not causing problems.